src/config/constantMessaging.ts

```diff
@@ -57,4 +57,10 @@ export const SSO_NOT_CONFIGURED_STATE_TEXTS = {
         ' Devtron uses Single Sign-On (SSO) to enable one-click login. Please set up an SSO login service before adding users.Go to SSO login services',
     linkText: 'Go to SSO login services',
     redirectLink: '/global-config/login-service',
+}
+
+export const ERR_MESSAGE_ARGOCD = 'Deployment pipeline cannot be attached to a pipeline being deleted.'
+
+export const LOCAL_STORAGE = {
+   ISDELTEDAPPFIRSTTIME : 'isDeletedAppFirstTime'
```

Comment:
simply name it to `isDeletedAppFirstTime` or `IsDeletedAppFirstTime`

Reply:
Removed this one as it was deprecated